exec/java-exec/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -278,3 +278,19 @@ SqlNode SqlRefreshMetadata() :
     }
 }
 
+/**
```

Comment:
What do you think about including this syntax directly in Calcite? @julianhyde, do you think this is generic enough to be shared or should we keep only in Drill code?